src/Infrastructure/LeanCode.SendGrid/SendGridLocalizedRazorMessage.cs

```diff
@@ -35,7 +35,7 @@ public void SetGlobalSubject(string subjectKey, object[]? subjectFormatArgs)
 
     internal override IEnumerable<string> GetTemplateNames(string templateBaseName)
     {
-        for (var c = Culture; c != CultureInfo.InvariantCulture; c = c.Parent)
+        for (var c = Culture; !Equals(c, CultureInfo.InvariantCulture); c = c.Parent)
```

Comment:
But why?

Reply:
Both will work as `CultureInfo.InvariantCulture` is static and will have the same reference, but `CultureInfo` does not implement `==` and `!=` operators which means it is reference comparison and it's probably better to use `Equals` when comparing `CultureInfo`.